### PR TITLE
Add Sailfish OS and Sailfish Browser to the detection list and tests

### PR DESCRIPTION
**Pull request by rainemak**

### CODE REVIEW

**Comment by rainemak** [comment]
Rebased...

**Comment by saukko** [comment]
@serbanghita any comments on this?

**Comment by serbanghita** [comment]
@rainemak thanks, will publish this in the next release! cc @saukko

File: Mobile_Detect.php [hunk 1]
```diff
@@ -1397,6 +1402,9 @@ public function mobileGrade()
             // Meego 1.2 - Tested on Nokia 950 and N9
             $this->is('MeeGoOS') ||
 
+            // Sailfish OS
+            $this->is('SailfishOS') ||
```
Comment (serbanghita) [inline Mobile_Detect.php]: info: mobileGrade is deprecated

Comment (rainemak) [reply Mobile_Detect.php]: ok. doesn't harm either.

**Pull request merged**